TI-reports/2024/2024-Q4-DEI-WG.md

```diff
@@ -0,0 +1,89 @@
+# 2024 Q4 DEI WG
+
+Chairs: [Marcela Melara](https://github.com/marcelamelara), [Yesenia Yser](https://github.com/Cyber-JiuJiteria) and [Jay White](https://github.com/camaleon2016)
+
+## Overview
+
+The DEI WG continues our mission to increase representation and strengthen
+the overall effectiveness of the cybersecurity workforce. The WG continues to
+see relatively low regular attendance in meetings, we are seeing increased
+engagement on Slack.
+
+Over the past quarter, we've made progress in these areas:
+
+* Our monthly Community Office Hours have been regularly attracting X+ attendees.
```

Comment:
what's the value of X here?

Reply:
Thanks for the reminder, I have the data now, so I'll replace X.

Reply:
done.